securedrop_client/gui/widgets.py

```diff
@@ -889,6 +889,22 @@ def show_no_source_selected_message(self):
         self.no_source_selected.show()
 
 
+class SourceListWidgetItem(QListWidgetItem):
+
+    def __lt__(self, other):
+        """
+        Used for ordering widgets by timestamp of last interaction.
+        """
+        lw = self.listWidget()
+        me = lw.itemWidget(self)
+        them = lw.itemWidget(other)
+        if me and them:
+            my_ts = arrow.get(me.source.last_updated)
+            other_ts = arrow.get(them.source.last_updated)
+            return my_ts < other_ts
+        return True
```

Comment:
nice implementation! 

cc @creviera for your awareness this impacts #967 - I think merging this and then as part of #967 modifying `SourceWidget.update` to also update a `last_updated` field makes sense. What do you think?

Reply:
that sounds good to me @redshiftzero